internal/pkg/backend/backend.go

```diff
@@ -0,0 +1,34 @@
+package backend
+
+import (
+	"fmt"
+
+	"github.com/merico-dev/stream/internal/pkg/backend/local"
+)
+
+var backends map[string]Backend
+
+func init() {
+	if backends == nil{
+		backends = map[string]Backend{
+			"local": local.NewLocal(""),
+		}
+	}
+}
+
+// Backend is used to persist data, it can be local file/etcd/s3/...
+type Backend interface {
+	// Read is used to reads data from persistent storage.
+	Read() ([]byte, error)
+	// Write is used to writes data to persistent storage.
+	Write(data []byte) error
+}
+
+// GetBackend will return a Backend by the given name.
+func GetBackend(name string) (Backend, error) {
```

Comment:
Add one simple test case for this function.

Reply:
Ok, I will try to add more unit tests within the month, same as above conversation.